src/scripts/main.js

```diff
@@ -22,7 +22,18 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key in data) {
+    const newList = document.createElement('ul');
+    const newElement = document.createElement('li');
```

Comment:
It's not good to use new in variables names.
```suggestion
    const list = document.createElement('ul');
    const item = document.createElement('li');
```